### PR TITLE
Employed recommendation of Issue #154

### PR DESCRIPTION
**Pull request by justgo129**
Issue #154

### CODE REVIEW

File: lib/Tuba/files/templates/figure/object.ttl.tut [hunk 1]
```diff
@@ -13,7 +13,7 @@
    dcterms:title "<%= $figure->title %>"^^xsd:string;
    cito:cites "<%= no_tbibs($figure->source_citation) %>"^^xsd:string;
    gcis:hasCaption "<%= no_tbibs($figure->caption) %>"^^xsd:string;
-   dcterms:RightsStatement "<%= $figure->usage_limits %>"^^xsd:string;
+   dcterms:Rights [ rdf:value "<%= $figure->usage_limits %>"^^xsd:string; ] ;
```
Comment (bduggan) [inline lib/Tuba/files/templates/figure/object.ttl.tut]: I think this should be rights, not Rights, right?

**Comment by justgo129** [comment]
right!  Stand by.

**Comment by bduggan** [comment]
Looks good, thanks.

**Comment by zednis** [comment]
closes #154

**Comment by justgo129** [comment]
This still looks open, would you mind reclosing?

**Comment by zednis** [comment]
For pull requests being merged is the same as being closed for a normal ticket.

**Comment by justgo129** [comment]
Understood.  Thanks for the heads-up, as that's a different approach than what the GCIS team has been using in the past.  We've merged, and then officially closed.  Either way works.

**Pull request merged**